pennylane/transforms/qcut/qcut.py

```diff
@@ -12,34 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Functions for performing quantum circuit cutting.
+Miscellaneous support for circuit cutting.
```

Comment:
We might need another PR in the future to rename this file. The only place Miscellaneous stuff should be is `utils.py` and even then its something we want to avoid overall. But I think for now this is fine !